atlp-rwanda/e-comm-team-amigos-bn
# chore(project structure): project setup #184581160

#### What does this PR do?
This PR creates initial project setup with empty boilerplate and dotenv configuration.
#### Description of tasks to be completed?

- setup the initial project structure
- create empty directory such as controllers, helpers, middleware for organizing the code and assets.
- add dotenv package to handle configuration information, allowing us to securely store sensitive data in environment variables.

#### How should this be manually tested?

- in your terminal run `git clone https://github.com/atlp-rwanda/e-comm-team-amigos-bn`  to clone this repository
- type `cd .\e-comm-team-amigos-bn\` to switch your working directory
- run`npm install` to install all dependencies that comes with this repository
- in your project root directory create`.env` file and initialize all necessary environment variables, don't hesitate to check .env.example to reference your self
- run `npm start` to start running node js engine and express server

#### Any background context you want to provide?
- always app will start running from `index.js` file located in src directory
#### #### What are the relevant pivotal tracker stories?
Story ID #184581160
